tox.ini

```diff
@@ -113,17 +103,38 @@ commands=
     sh -c "conda run -n poli__rasp python -m pip install -e ."
     pytest {tty:--color=yes} -v -m "not slow and poli__rasp" {posargs}
 
-[testenv:poli-rmf-py39]
-description = run the tests with pytest on the dockstring environment for poli
-basepython = python3.9
+[testenv:poli-rmf-py310]
+description = run the tests with pytest on the rmf environment for poli
+basepython = python3.10
+wheel_build_env = .pkg
+deps=
+    {[testenv]deps}
+    -r requirements.txt
+    -e.[rmf]
+commands=
+    pytest {tty:--color=yes} -v -m 'not slow and poli__rmf' {posargs}
+
+[testenv:poli-ehrlich-holo-py310]
+description = run the tests with pytest on the ehrlich environment for poli
+basepython = python3.10
+wheel_build_env = .pkg
+deps=
+    {[testenv]deps}
+    -r requirements.txt
+    -e.[ehrlich_holo]
+commands=
+    pytest {tty:--color=yes} -v -m 'not slow and poli__ehrlich_holo' {posargs}
+
+[testenv:poli-rosetta_energy-py310]
+description = run the tests with pytest on the rosetta_energy environment for poli
+basepython = python3.10
 wheel_build_env = .pkg
 deps=
     {[testenv]deps}
     -r requirements.txt
     -e.
 commands=
-    sh -c 'if conda info --envs | grep -q poli__rmf; then echo "poli__rmf already exists"; else conda env create -f ./src/poli/objective_repository/rmf_landscape/environment.yml; fi'
-    sh -c "conda run -n poli__rmf python -m pip uninstall -y poli"
-    sh -c "conda run -n poli__rmf python -m pip install -e ."
-    sh -c "conda run -n poli__rmf pip install pytest"
-    sh -c "conda run -n poli__rmf pytest -v -m 'not slow and poli__rmf'"
\ No newline at end of file
+    sh -c 'if conda info --envs | grep -q poli__rosetta_energy; then echo "poli__rosetta_energy already exists"; else conda env create -f ./src/poli/objective_repository/rosetta_energy/environment.yml; fi'
```

Comment:
Is it absolutely necessary to have `conda` as a dependency here? Or do you think it would run if we replace `-e.` with `-e.[rosetta_energy]` as per the optional dependencies in the `pyproject.toml`?

Reply:
Possible via [this installer](https://pypi.org/project/pyrosetta-installer/) but not recommended procedure. Prefer to download pre-compiled package via conda unfortunately.